qemu/tests/cfg/tpm_check_buffer_size.cfg

```diff
@@ -0,0 +1,8 @@
+- tpm_check_buffer_size:
+  virt_test_type = qemu
+  type = tpm_check_buffer_size
+  start_vm = no
+  repo_url = https://github.com/stefanberger/libtpms
+  test_case = tpm2_cve-2023-1017.c;
+  test_case += tpm2_cve-2023-1018.c
+  build_execute_cmd = gcc tests/%s -ltpms && ./a.out
```

Comment:
Consider using `-o tpm2_cve-2023-1017/tpm2_cve-2023-1018` to replace `a.out`?

Reply:
Updated.